libnvme/src/nvme/ioctl.h

```diff
@@ -4410,7 +4410,7 @@ nvme_init_dsm(struct nvme_passthru_cmd *cmd,
 	cmd->nsid	= nsid;
```

Comment:
The docs for the arguments got a bit shuffled around.
Should be something like:
```
...
 * @idr:	DSM Integral Dataset for Read attribute
 * @idw:	DSM Integral Dataset for Write attribute
 * @ad: 	DSM Deallocate attribute
...
```